### PR TITLE
IFTAQCAB-55 Extract work with table to separate class

### PR DESCRIPTION
**Pull request by promahos**

### CODE REVIEW

File: src/main/java/com/softserveinc/ita/models/EntityTable.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package com.softserveinc.ita.models;
```
[the remark this answers is not in the document]
Comment (promahos) [reply src/main/java/com/softserveinc/ita/models/EntityTable.java]: Agree, I placed it there by mistake.
Moved to pageobjects

File: src/test/java/com/softserveinc/ita/SubjectsTest.java [hunk 2]
```diff
@@ -74,18 +74,19 @@ public void verifyAddingNewSubject() {
 
         new SubjectStep().openAndFillSubjectFields(getValidSubject());
 
-        var isAddedAtTheEnd = new AddingSubjectModal()
-                .addNewSubject()
-                .switchToLastPageOfTable()
-                .getNamesOfSubjects()
+        var newSubjectPage = new AddingSubjectModal().addNewSubject();
+        newSubjectPage.getTable().goToTablePage("last");
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/SubjectsTest.java:78]: chain

```
newSubjectPage
                .getTable()
                .goToTablePage("last");
```

File: src/test/java/com/softserveinc/ita/SubjectsTest.java [hunk 3]
```diff
@@ -74,18 +74,19 @@ public void verifyAddingNewSubject() {
...
+
+        var isAddedAtTheEnd = newSubjectPage.getNamesOfSubjects()
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/SubjectsTest.java:80]: same

File: src/test/java/com/softserveinc/ita/SubjectsTest.java [hunk 4]
```diff
@@ -74,18 +74,19 @@ public void verifyAddingNewSubject() {
...
                 .contains(subjectName);
 
         assertThat(isAddedAtTheEnd)
                 .as("New subject should be displayed at the end of table")
                 .isTrue();
 
+        subjectsPage.getTable().goToTablePage("first");
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/SubjectsTest.java:87]: same

**Review by Nikodych — CHANGES_REQUESTED** [review]

**Review by Nikodych — APPROVED** [review]

**Pull request merged**